Expanded-info.md

```diff
@@ -0,0 +1,48 @@
+#PR for discussion
+
+## Problem description
+
+Today, if you use JSON generated from *.vspec information on whether a branch existed in *.vspec or comes from
+instance expansion (e.g. DriverSide, PassengerSide) is lost. This makes it difficult if you want to handle instantiation
+differently in your tool, i.e. you want a representation similar to what exist in *.vspec files.
+
+This PR prototypes two methods, and is intended for discussion
+
+## Alternative 1: Annotate that a branch is expanded
+
+In this PR an example for JSOn is shown, running as today:
+
+```
+vss-tools/vspec2json.py -I ./spec --uuid -u ./spec/units.yaml --json-pretty ./spec/VehicleSignalSpecification.vspec expand.json
+```
+
+...gives an extra output `expanded:true` for branches that has been expanded.
+
+I.e. branches like `Vehicle.Cabin.Seat.Row2.PassengerSide` will have `expanded:true`, while regular branches like
+`Vehicle.Cabin` and `Vehicle.Cabin.Seat` will not have any `expanded` attribute.
+
```

Comment:
Disadvantages would be:
 *  The detection of instances is possible, in principle. But it requires extra effort.
 *  The detection might face some limits, when it comes to instances that contain attributes or signals valid for that single instance (or more precisely: a limited set of instances)
 * Digital.Auto is using a json-based VSS catalogue which is "not aware" of instances. There, the "expanded" attribute would need to be maintained manually, which probably leads to additional error cases.